ocs/ocp.py

```diff
@@ -259,6 +259,34 @@ def wait_for_resource(
 
         return False
 
+    def describe(
+        self, resource_name='', out_yaml_format=True, selector=None,
+        all_namespaces=False
+    ):
+        """
+        Get command - 'oc describe <kind> <resource>'
+
+        Args:
+            resource_name (str): The resource name to fetch
+            out_yaml_format (bool): Adding '-o yaml' to oc command
+            selector (str): The label selector to look for
+            all_namespaces (bool): Equal to oc get <resource> -A
+
+        Example:
+            describe(pod, csi-cephfsplugin-provisioner-0)
+
+        Returns:
+            dict: Dictionary represents a returned yaml file
+        """
+        command = f"get {self.kind} {resource_name}"
+        if all_namespaces and not self.namespace:
+            command += "-A"
```

Comment:
There are missing spaces between command line arguments

Reply:
removed this block since I am not going to use that